Do you know of an nf-core publication that we're missing?
Please [make a pull-request](https://github.com/nf-core/nf-co.re/blob/master/markdown/publications.md).

# The nf-core project

Main publication for the main nf-core paper, describing the community and framework:

<!-- pub-stats 10.1038/s41587-020-0439-x -->
> **The nf-core framework for community-curated bioinformatics pipelines**
>
> Philip Ewels, Alexander Peltzer, Sven Fillinger, Harshil Patel, Johannes Alneberg, Andreas Wilm, Maxime Ulysse Garcia, Paolo Di Tommaso & Sven Nahnsen
>
> [_Nat Biotechnology_ (2020)](https://www.nature.com/articles/s41587-020-0439-x);
> doi: [10.1038/s41587-020-0439-x](https://doi.org/10.1038/s41587-020-0439-x)
>
> ReadCube: [View full text](https://rdcu.be/b1GjZ)

Preprint for the main nf-core paper (significantly different from the final Nature Biotech paper):

<!-- pub-stats 10.1101/610741 -->
> **_nf-core:_ Community curated bioinformatics pipelines**
>
> Philip Ewels, Alexander Peltzer, Sven Fillinger, Johannes Alneberg, Harshil Patel, Andreas Wilm, Maxime Ulysse Garcia, Paolo Di Tommaso, Sven Nahnsen
>
> [_bioRxiv_ 610741](https://www.biorxiv.org/content/10.1101/610741v3);
> doi: [10.1101/610741](https://doi.org/10.1101/610741)

Article in _Nature_ describing scientific workflows (including nf-core):

<!-- pub-stats 10.1038/d41586-019-02619-z -->
> **Workflow systems turn raw data into scientific knowledge**
>
> Jeffrey M. Perkel
>
> [_Nature_ **573**, 149-150 (2019)](https://www.nature.com/articles/d41586-019-02619-z);
> doi: [10.1038/d41586-019-02619-z](https://doi.org/10.1038/d41586-019-02619-z)

# Pipeline publications

### [nf-core/ampliseq](https://nf-co.re/ampliseq)

<!-- pub-stats 10.3389/fmicb.2020.550420 -->
> **Interpretations of Environmental Microbial Community Studies Are Biased by the Selected 16S rRNA (Gene) Amplicon Sequencing Pipeline**
>
> Daniel Straub, Nia Blackwell, Adrian Langarica-Fuentes, Alexander Peltzer, Sven Nahnsen and Sara Kleindienst
>
> [_Frontiers in Microbiology_ (2020)](https://www.frontiersin.org/articles/10.3389/fmicb.2020.550420/full)
> doi: [10.3389/fmicb.2020.550420](https://doi.org/10.3389/fmicb.2020.550420)

### [nf-core/coproid](https://nf-co.re/coproid)

<!-- pub-stats 10.7717/peerj.9001 -->
> **CoproID predicts the source of coprolites and paleofeces using microbiome composition and host DNA content**
>
> Maxime Borry, Bryan Cordova, Angela Perri, Marsha Wibowo, Tanvi Prasad Honap, Jada Ko, Jie Yu, Kate Britton, Linus Girdland-Flink, Robert C. Power, Ingelise Stuijts, Domingo C. Salazar-Garcia, Courtney Hofman, Richard Hagan, Thérèse Samdapawindé Kagoné, Nicolas Meda, Hélène Carabin, David Jacobson, Karl Reinhard, Cecil Lewis, Aleksandar Kostic, Choongwon Jeong, Alexander Herbig, Alexander Hübner, Christina Warinner
>
> [_PeerJ_, 9001 (2020)](https://peerj.com/articles/9001);
> doi: [10.7717/peerj.9001](https://doi.org/10.7717/peerj.9001)

### [nf-core/diaproteomics](https://nf-co.re/diaproteomics)

<!-- pub-stats 10.1021/acs.jproteome.1c00123 -->
> **DIAproteomics: A Multifunctional Data Analysis Pipeline for Data-Independent Acquisition Proteomics and Peptidomics**
>
> Leon Bichmann, Shubham Gupta, George Rosenberger, Leon Kuchenbecker, Timo Sachsenberg, Phil Ewels, Oliver Alka, Julianus Pfeuffer, Oliver Kohlbacher, and Hannes Röst
>
> [J. Proteome Res. (2021)](https://pubs.acs.org/doi/abs/10.1021/acs.jproteome.1c00123)
> doi: [10.1021/acs.jproteome.1c00123](https://doi.org/10.1021/acs.jproteome.1c00123)

### [nf-core/eager](https://nf-co.re/eager)

<!-- pub-stats 10.7717/peerj.10947 -->
> **Reproducible, portable, and efficient ancient genome reconstruction with nf-core/eager**
>
> James A. Fellows Yates, Thiseas C. Lamnidis, Maxime Borry, Aida Andrades Valtueña, Zandra Fagernäs, Stephen Clayton, Maxime U. Garcia, Judith Neukamm, Alexander Peltzer
>
> [_PeerJ_, 9:e10947 (2021)](https://peerj.com/articles/10947/);
> doi: [10.7717/peerj.10947](https://doi.org/10.7717/peerj.10947)

### [nf-core/imcyto](https://nf-co.re/imcyto)

<!-- pub-stats 10.1101/2021.02.02.429358 -->
> **Characterisation of tumour immune microenvironment remodelling following oncogene inhibition in preclinical studies using an optimised imaging mass cytometry workflow.**
>
> Febe van Maldegem, Karishma Valand, Megan Cole, Harshil Patel, Mihaela Angelova, Sareena Rana, Emma Colliver, Katey Enfield, Nourdine Bah, Victoria Tsang, Edurne Mugarza, Christopher Moore, Philip Hobson, Dina Levi, Miriam Molina, Charles Swanton,  View ORCID ProfileJulian Downward.
>
> [_BiorXiV_, 10.1101/2021.02.02.429358 (2021)](https://www.biorxiv.org/content/10.1101/2021.02.02.429358v1);
> doi: [10.1101/2021.02.02.429358](https://doi.org/10.1101/2021.02.02.429358)

### [nf-core/lncpipe](https://nf-co.re/lncpipe)

<!-- pub-stats 10.1016/j.jgg.2018.06.005 -->
> **LncPipe: A Nextflow-based pipeline for identification and analysis of long non-coding RNAs from RNA-Seq data.**
>
> Qi Zhao, Yu Sun, Dawei Wang, Hongwan Zhang, Kai Yu, Jian Zheng, Zhixiang Zuo.
>
> [_Journal of Genetics and Genomics_ **20**;45(7):399-401 (2018)](https://www.sciencedirect.com/science/article/pii/S1673852718301176)
> doi: [10.1016/j.jgg.2018.06.005](https://doi.org/10.1016/j.jgg.2018.06.005)

### [nf-core/mhcquant](https://nf-co.re/mhcquant)

<!-- pub-stats 10.1021/acs.jproteome.9b00313 -->
> **MHCquant: Automated and Reproducible Data Analysis for Immunopeptidomics**
>
> Leon Bichmann, Annika Nelde, Michael Ghosh, Lukas Heumos, Christopher Mohr, Alexander Peltzer, Leon Kuchenbecker, Timo Sachsenberg, Juliane S. Walz, Stefan Stevanović, Hans-Georg Rammensee, and Oliver Kohlbacher
>
> [_Journal of Proteome Research_ **18**(11), 3876-3884 (2019)](https://pubs.acs.org/doi/10.1021/acs.jproteome.9b00313)
> doi: [10.1021/acs.jproteome.9b00313](https://doi.org/10.1021/acs.jproteome.9b00313)

### [nf-core/nanoseq](https://nf-co.re/nanoseq)

<!-- pub-stats 10.1101/2021.04.21.440736 -->
> **A systematic benchmark of Nanopore long read RNA sequencing for transcript level analysis in human cell lines**
>
> Ying Chen, Nadia M. Davidson, Yuk Kei Wan, Harshil Patel, Fei Yao, Hwee Meng Low, Christopher Hendra, Laura Watten, Andre Sim, Chelsea Sawyer, Viktoriia Iakovleva, Puay Leng Lee, Lixia Xin, Hui En Vanessa Ng, Jia Min Loo, Xuewen Ong, Hui Qi Amanda Ng, Jiaxu Wang, Wei Qian Casslynn Koh, Suk Yeah Polly Poon, Dominik Stanojevic, Hoang-Dai Tran, Kok Hao Edwin Lim, Shen Yon Toh, Philip Andrew Ewels, Huck-Hui Ng, N.Gopalakrishna Iyer, Alexandre Thiery, Wee Joo Chng, Leilei Chen, Ramanuj DasGupta, Mile Sikic, Yun-Shen Chan, Boon Ooi Patrick Tan, Yue Wan, Wai Leong Tam, Qiang Yu, Chiea Chuan Khor, Torsten Wüstefeld, Ploy N. Pratanwanich, Michael I. Love, Wee Siong Sho Goh, Sarah B. Ng, Alicia Oshlack, Jonathan Göke, SG-NEx consortium
>
> [_bioRxiv_ 610741](https://www.biorxiv.org/content/10.1101/2021.04.21.440736v1);
> doi: [10.1101/610741](https://doi.org/10.1101/2021.04.21.440736)

### [nf-core/sarek](https://nf-co.re/sarek)

<!-- pub-stats 10.12688/f1000research.16665.2 -->
> **Sarek: A portable workflow for whole-genome sequencing analysis of germline and somatic variants** [version 2; peer review: 2 approved].
>
> Maxime Garcia, Szilveszter Juhos, Malin Larsson, Pall I. Olason, Marcel Martin, Jesper Eisfeldt, Sebastian DiLorenzo, Johanna Sandgren, Teresita Díaz De Ståhl, Philip Ewels, Valtteri Wirta, Monica Nistér, Max Käller, Björn Nystedt
>
> [_F1000Research_, 9:63 (2020)](https://f1000research.com/articles/9-63/v2)
> doi: [10.12688/f1000research.16665.2](https://doi.org/10.12688/f1000research.16665.2)
